ome-demoserver.yml

```diff
@@ -119,15 +119,16 @@
         databases: ["{{ vault.omero_server_dbname }}"]
 
     - role: openmicroscopy.omero-server
-      omero_server_release: 5.4.3
+      omero_server_release: 5.4.4
       omero_server_dbuser: "{{ vault.omero_server_db_user }}"
       omero_server_dbname:  "{{ vault.omero_server_dbname }}"
       omero_server_dbpassword: "{{ vault.omero_server_dbpassword }}"
       omero_server_rootpassword: "{{ vault.omero_server_rootpassword }}"
       omero_server_systemd_limit_nofile: 16384
+      omero_server_system_user_manage: False
```

Comment:
@jburel - we do manage the user with this role for `ome-demoserver`, just not `ome-dundeeomero`.

Reply:
ok rolling that back